pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1925,6 +2056,37 @@ func buildHTTPConnectionManager(pluginParams *plugin.InputParams, httpOpts *http
 	return connectionManager
 }
 
+func buildThriftRatelimit(rlsCluster, domain string) *thrift_ratelimit.RateLimit {
+	thriftRateLimit := &thrift_ratelimit.RateLimit{
+		Domain:          domain,
+		Timeout:         ptypes.DurationProto(20 * time.Millisecond),
```

Comment:
Seems potentially pretty low as a default?

Reply:
Added to configuration